modules/nw-dns-forward.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-forward_{context}"]
+= Using DNS forwarding
+
+You can use DNS forwarding to override the forwarding configuration identified in `etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone.
+
+.Procedure
+
+. Modify the DNS Operator object named `default`:
++
+----
+$ oc edit dns.operator/default
+----
+.. This allows the Operator to create and update the ConfigMap named `dns-default` with additional server configuration blocks based on `Server`. If none of the servers has a zone that matches the query, then name resolution falls back to the name servers that are specified in `/etc/resolv.conf`.
```

Comment:
Consider whether or not this needs to be a step (1a) or if it can just be included as text under step 1. Kind of like how step 2 does it. Just good to be consistent, and since there's only 1 substep there, probably not needed.